library/system/src/lib/yast2/execute.rb

```diff
@@ -30,12 +30,21 @@ module Yast
   # as the backend, but adds support for chrooting during the installation.
   # It also globally switches the default Cheetah logger to
   # {http://www.rubydoc.info/github/yast/yast-ruby-bindings/Yast%2FLogger Y2Logger}.
-  class Execute
+  class ExecuteClass
```

Comment:
But it is not possible as Execute is now object and not namespace. Goal is to keep working existing API and allow chaining of commands.

Reply:
Well,  `Execute::Result` is not even necessary. We can simply continue with `on_target` and `locally` as class methods, and have them return instances of the same class so that you can call methods on that.

I do feel strongly about not allowing ExecuteClass.